### ISSUE TITLE
Where can I get dataset?

### ISSUE THREAD

**Issue by 17011813** [phaphuang/MAML-VAE#1]
Good Morning!

I want to run your project.
How can I get the full code and datasets?

Also I want to know torch-gpu and python version that you used :)

Thank you~!

**Comment by phaphuang** [comment]
Sorry this project is quite old. I am not sure I successfully completed the project at that time. Original idea and code I got from this project, please refer to following urls:
https://medium.com/@utkarshjp7/generating-images-with-few-shot-meta-learning-25bf1d301ab0
https://github.com/usc599resl/mlgm/

**Comment by 17011813** [comment]
Thanks for the fast response.

I already tried the code that you mentioned above, but that code had some theoretical problems.

1. There is a problem of gradient update several times during a meta-test.
2. Meta-learning requires training of multiple tasks per iteration(epoch), but there is a problem of executing only one task per iteration.

Because of these 2 issues, I try to find another code and I got your repository yesterday!
And I think your code is different from that one. Yours is more accurate and correct :)

So, did you know and solve these two problems of the above code?
Then did you change your code by referring to the above project?


*p.s. I saw your google scholar and you did research about meta-learning.
Is this paper is about this project? https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9424414

Also, What's the difference between this project? https://github.com/phaphuang/generative-few-shot-figr-vae
